### ISSUE TITLE
[Windows] applying Patch does not work across machines

### ISSUE THREAD

**Issue by domiSchenk** [ds300/patch-package#124]
We are using yarn and somehow, patch-package works only on the machine that created the patch.

```
Node: 10.13.0 (dev) / 10.15.1 (Teamcity)
Yarn: 1.13.0
patch-package: 6.0.2
```
Message in console:
```
patch-package 6.0.2
Applying patches...

**ERROR** Failed to apply patch for package @turf/difference at path

    node_modules\@turf\difference

  This error was caused because patch-package cannot apply the following patch file:

    patches/@turf+difference+6.0.2.patch

  Try removing node_modules and trying again. If that doesn't work, maybe there was
  an accidental change made to the patch file? Try recreating it by manually
  editing the appropriate files and running:

    patch-package @turf/difference

  If that doesn't work, then it's a bug in patch-package, so please submit a bug
  report. Thanks!

    https://github.com/ds300/patch-package/issues

```
Patch file: @turf+difference+6.0.2.patch
```
diff --git a/node_modules/@turf/difference/index.js b/node_modules/@turf/difference/index.js
index 89ebe87..a939e3b 100644
--- a/node_modules/@turf/difference/index.js
+++ b/node_modules/@turf/difference/index.js
@@ -80,4 +80,4 @@ function removeEmptyPolygon(geom) {
     }
 }
 
-module.exports = difference;
+export default difference;
```

We have this problem with every patch does not matter if its turf/difference or an other one.

could this be an white space problem?
let me know if you need more information

**Comment by ds300** [comment]
Hi! :wave: Thanks for the report. It's unlikely to be a whitespace problem unless teamcity is running windows. Not sure what the error could be. I'll investigate this. Thanks for providing node versions!

**Comment by domiSchenk** [comment]
Teamcity is also running on windows :-D

**Comment by ds300** [comment]
Try making sure that teamcity's git config has the same CRLF settings as you have locally.

**Comment by GerrardAquilon** [comment]
I've got the same problem here, the patch works fine on my computer but does not work on another. Both on windows.

**Comment by domiSchenk** [comment]
> Try making sure that teamcity's git config has the same CRLF settings as you have locally.

it is not whitespace :-(

i run ` git config --global core.autocrlf true` on teamcity and my PC.

what i find strange is, i used version 6.0.0-9 and then i worked on all machines.

**Comment by franksmule** [comment]
Same issue here, created patch on one windows box, applying fails on another.

Going to version 6.0.0-9 fixes the issue.

**Comment by ds300** [comment]
Can anyone share a repo with a patch file that definitely fails with 6.0.x? I'm going to try to reproduce this in VirtualBox.

**Comment by edcoreweb** [comment]
On my env works until 6.0.0.14 inclusively.
6.0.0-15 fails

**Comment by ds300** [comment]
Thank you so much @edcoreweb for the reproduction :heart:

I was able to fix the issue affecting that project at least. It should be available in 6.0.4 any minute now.

If anyone else is still having problems feel free to open another issue :pray:

**Comment by nithin-samudrala** [comment]
I am facing the same issue I am using the 6.4.7 version can anyone help me
